modules/confidential-data/versions.tf

```
@@ -31,12 +31,12 @@ terraform {
 
     null = {
       source  = "hashicorp/null"
-      version = "~> 2.1"
+      version = "3.2.0"
```

Avoid breaking change

```suggestion
      version = ">= 2.1"
```